src/selectors/per-thread/markers.ts

```diff
@@ -695,9 +685,52 @@ export function getMarkerSelectorsPerThread(
         )
     );
 
+    const getCommittedRangeMinMax: Selector<{
+      minNumber: number;
+      maxNumber: number;
+    }> = createSelector(
+      getCollectedCustomMarkerSamples,
+      getCommittedRangeMarkerSampleRange,
+      (collectedSamples, sampleRange) => {
+        const [sampleStart, sampleEnd] = sampleRange;
+        const { numbersPerLine } = collectedSamples;
+
+        // Handle edge case where there are no samples in range
+        if (sampleStart >= sampleEnd) {
+          return { minNumber: 0, maxNumber: 0 };
+        }
+
+        let minNumber = Infinity;
+        let maxNumber = -Infinity;
+
+        for (
+          let sampleIndex = sampleStart;
+          sampleIndex < sampleEnd;
+          sampleIndex++
+        ) {
+          for (
+            let graphIndex = 0;
+            graphIndex < numbersPerLine.length;
+            graphIndex++
+          ) {
+            const val = numbersPerLine[graphIndex][sampleIndex];
+            if (val < minNumber) {
+              minNumber = val;
+            }
+            if (val > maxNumber) {
+              maxNumber = val;
+            }
```

Comment:
Existing issue, but: It's confusing me a bit that we're looking at each graph in isolation. Don't we stack the values up vertically within a graph track? Wouldn't we need the bounds of the *sum* of the graph values at each x position, to scale things correctly?